### PR TITLE
{devel}[foss/2018a] SWIG 3.0.12 w/ Python 2.7.14 + 3.6.4

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

File: easybuild/easyconfigs/p/PCRE/PCRE-8.41-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name = 'PCRE'
+version = '8.41'
+
+homepage = 'http://www.pcre.org/'
+description = """
+ The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
+ and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
```
Comment (boegel) [inline easybuild/easyconfigs/p/PCRE/PCRE-8.41-foss-2018a.eb]: @RvDijk Please just remove this easyconfig, we already have `PCRE` `8.41` with `GCCcore/6.4.0`, which is good enough and can serve as dep for SWIG with both `foss/2018a` & `intel/2018a`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2666.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d271bfa43610f32c726b8c5b5c420dad for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3e716ec9b6e81eb588e9e00b70bbc66c for a full test report.

File: easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-foss-2018a-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '3.0.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```
Comment (boegel) [inline easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-foss-2018a-Python-2.7.14.eb]: @RvDijk should just be `toolchainopts = {'pic': True}`? (same in other `SWIG` easyconfig)

Comment (RvDijk) [reply easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-foss-2018a-Python-2.7.14.eb]: @boegel I got these from the `foss 2016b` or comparable version.. Will remove!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2666.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2486aafcd77ee0f4bfe97b1961109715 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1af4bc9787a744c7d79c4c58b3722610 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**